src/games/strategy/engine/history/SerializationWriter.java

```diff
@@ -0,0 +1,7 @@
+package games.strategy.engine.history;
+
+import java.io.Serializable;
+
+public interface SerializationWriter extends Serializable {
```

Comment:
@DanVanAtta Java dislikes the fact that auxiliary classes are used in the same package.
If this interface would only be used inside History.java this wouldn't be a problem